### PR TITLE
use composite build-and-push-action

### PR DESCRIPTION
**Pull request by zmiklank**
- rebuild images every wednesday
- enable action only for sclorg repositories

### CODE REVIEW

File: .github/workflows/build-and-push.yml [hunk 1]
```diff
@@ -1,79 +1,39 @@
...
+      - cron: '0 1 * * 3'
 
 jobs:
   build-and-push:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
       matrix:
-        version: [7.3, 7.4, 8.0]
+        dockerfile_path: ["7.3", "7.4", "8.0"]
```
Comment (phracek) [inline .github/workflows/build-and-push.yml:15]: Version '8.0' is not part of the repository yet ;)

**Review by phracek — CHANGES_REQUESTED** [review]
Just version '8.0' is not part of upstream. It will be as soon as https://github.com/sclorg/s2i-php-container/pull/347 is merged.

File: .github/workflows/build-and-push.yml (hunk 1 above)
Comment (zmiklank) [reply .github/workflows/build-and-push.yml:15]: Interesting, because it was in the build-and-push even before my proposal. should I remove it?

Comment (phracek) [reply .github/workflows/build-and-push.yml:15]: @zmiklank Please remove version 8.0. It is not part of `master` branch yet ;)

**Review by phracek — CHANGES_REQUESTED** [review]

File: .github/workflows/build-and-push.yml (hunk 1 above)
Comment (zmiklank) [reply .github/workflows/build-and-push.yml]: Done, sorry for the delay.

**Review by phracek — APPROVED** [review]
LGTM

**Pull request merged**